src/components/scenes/Loans/LoanDashboardScene.tsx

```diff
@@ -146,11 +146,12 @@ export const LoanDashboardScene = (props: Props) => {
           <Space around>
             <FillLoader />
           </Space>
-        ) : null}
-        <TouchableOpacity onPress={handleAddLoan} style={styles.addButtonsContainer}>
-          <Ionicon name="md-add" style={styles.addItem} size={theme.rem(1.5)} color={theme.iconTappable} />
-          <EdgeText style={[styles.addItem, styles.addItemText]}>{s.strings.loan_new_loan}</EdgeText>
-        </TouchableOpacity>
+        ) : (
+          <TouchableOpacity onPress={handleAddLoan} style={styles.addButtonsContainer}>
+            <Ionicon name="md-add" style={styles.addItem} size={theme.rem(1.5)} color={theme.iconTappable} />
+            <EdgeText style={[styles.addItem, styles.addItemText]}>{s.strings.loan_new_loan}</EdgeText>
+          </TouchableOpacity>
+        )}
```

Comment:
I see the race condition we're trying to solve here. User can start the loan creation process before user has queried the network which means the user has the chance of completing the create workflow after the query finishes and a loan account is created.

This solve this problem at the tradeoff that the UI is stuck until all wallets have synced with the network. This blocking UX is not going to be reasonable for accounts with many wallets.

An alternative solution would be to check for loan account at the LoanCreateConfirmationScene to determine whether a `createLoanAccount` or `updateLoanAccount` action is to be dispatched. This covers our edge case with a the fallback behavior of "create workflow does update" which we know will work. This will avoid the expensive tradeoff.